agent/pkg/containerbuilder/container_builder.go

```diff
@@ -295,7 +305,16 @@ func (dc *dockerContainerBuilder) Start() (bool, error) {
 	}
 }
 
-func portListToNatBinding(portRanges []v1.PortRangeBinding, portList []v1.PortBinding) map[nat.Port][]nat.PortBinding {
+func execHooks(dc *DockerContainerBuilder, hooks []LifecycleFunc) {
+	for _, hook := range hooks {
+		err := hook(dc.ctx, dc.client, dc.containerName, &dc.containerID, dc.mountList, dc.logger)
+		if err != nil {
+			logWrite(dc, fmt.Sprintln("Container post-start hook error: ", err))
```

Comment:
this can by any hook, not just **post-start**.

Reply:
Fixed :)